_layouts/frontpage.html

```diff
@@ -162,25 +162,26 @@ <h2>Currently working on...</h2>
                     {% for project in site.projects %}
                       {% if project.hide-from-homepage != true %}
                       <a href="/projects.html#{{ project.label }}" class="project">
-                        {% if project.status == "Contributors Welcome!" %}
```

Comment:
Why were those changes necessary? Are they just cleanup? If yes, it would be good to put them in a separate commit, at least.